File: web-app/online-warehouse/src/app/write-off-act/dto/placement-write-off-act-goods.dto.ts
import {WriteOffTypeEnum} from "./enum/write-off-type.enum";
import {PlacementGoodsDto} from "../../warehouse/dto/placement-goods.dto";

export class PlacementWriteOffActGoodsDto {
  placementGoods: PlacementGoodsDto;
  writeOffType: WriteOffTypeEnum;
  amount: number;
}

File: core/src/main/java/by/itechart/writeoffact/dto/PlacementWriteOffActGoodsDto.java
package by.itechart.writeoffact.dto;

import by.itechart.warehouse.dto.PlacementGoodsDto;
import by.itechart.writeoffact.enums.WriteOffType;
import lombok.Data;

@Data
public class PlacementWriteOffActGoodsDto {
    private PlacementGoodsDto placementGoods;
    private WriteOffType writeOffType;
    private Integer amount;
}
